src/app/pdf-viewer/pdf-viewer.component.ts

```diff
@@ -573,7 +570,14 @@ export class PdfViewerComponent
       });
     }
 
-    this.updateSize();
+    if (!this.pdfViewer._pages?.length) {
+      // the first time we wait until pages init
+      this.pageInitialized.subscribe(() => {
+        this.updateSize();
+      })
```

Comment:
Potentially a bug, as it is subscribing on every re-render

Reply:
Something like this to unsubscribe once initialized?
```ts
const sub = this.pageInitialized.subscribe(() => {
  this.updateSize();
  sub.unsubscribe();
});
```

Reply:
Yea good point. I first did a boolean check but changed to be more like the suggestion